docs/Agent/user-guide/man_page.md

```diff
@@ -333,107 +265,205 @@ pbench-register-tool-trigger, Registers the tool trigger
 
 **OPTIONS**  
--C, --config PATH  
-Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable,if defined) [required]
+`--user`  
+This option value is required if not provided by the
+'PBENCH_USER' environment variable; otherwise, a value provided
+on the command line will override any value provided by the
+environment.
 
---tool-group  
-The tool group to use for the list of tools
+`--controller`
+This option may be used to override the value
+provided by the 'PBENCH_CONTROLLER' environment variable; if
+neither value is available, the result of 'hostname -f' is used.
+(If no value is available, the command will exit with an error.)
 
---iteration-list  
-A file containing a list of iterations to run for the provided script;
-the file should contain one iteration per line, with a leading '#' (hash) character used for comments, blank lines are ignored; each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script;  
-_NOTE: --iteration-list is not compatible with --use-tool-triggers_
+`--prefix`  
+This option allows the user to specify an optional
+directory-path hierarchy to be used when displaying the result
+tar balls on the pbench server.
 
---sysinfo STR,[STR]  
-comma-separated values of system information to be collected; available: default, none, all, ara, block, insights, kernel_config, libvirt, security_mitigations, sos, stockpile, topology
+`--show-server`  
+This will not move any results, but resolve and
+then display the pbench server destination for results.
 
---pbench-pre STR  
-Path to the script which will be executed before tools are started  
-_NOTE: --pbench-pre is not compatible with --use-tool-triggers_
+`--xz-single-threaded`  
+This will force the use of a single
+thread for locally compressing the result tar balls.
 
---pbench-post STR  
-Path to the script which will be executed after tools are stopped and postprocessing is complete  
-_NOTE: --pbench-post is not compatible with --use-tool-triggers_
+`--help`  
+Show this message and exit.
 
---use-tool-triggers  
-Use tool triggers instead of normal start/stop around script;  
-_NOTE: --use-tool-triggers is not compatible with --iteration-list,--pbench-pre, or --pbench-post_
+---
 
---no-stderr-capture  
-Do not capture the stderr of the script to the result.txt file
+##### pbench-copy-results
+
+---
+
+**NAME**  
+`pbench-copy-results`, Copies result tarball to the 0.69 Pbench Server.
+
+**SYNOPSIS**  
+`pbench-copy-results [--help] --user=<user> [--controller=<controller>] [--prefix=<path>][--xz-single-threaded] [--show-server]`
+
+**DESCRIPTION**
+Push the benchmark result to the Pbench Server without removing it from the local host .We use id_rsa when pushing to a 0.69 Pbench Server.
```

Comment:
Line 401 is missing whatever magic the other subsection headers have which causes the subsequent text to start on a new line.

Also, the spacing around the first period is incorrect.

Also, `id_rsa` should be a full path (in monospace font) to be consistent with `pbench-move-results`.

The use of `We` here is a change in voice/tone.  I would recommend going with something like, "This command uses...", instead.